engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetFacesToCull.java

```diff
@@ -15,34 +15,34 @@
  */
 package org.terasology.rendering.dag.stateChanges;
 
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
 import org.lwjgl.opengl.GL11;
 import static org.lwjgl.opengl.GL11.GL_BACK;
 import static org.lwjgl.opengl.GL11.GL_FRONT;
 import static org.lwjgl.opengl.GL11.GL_FRONT_AND_BACK;
 import org.terasology.rendering.dag.RenderPipelineTask;
 import org.terasology.rendering.dag.StateChange;
-import org.terasology.rendering.dag.tasks.SetWhatFacesToCullTask;
+import org.terasology.rendering.dag.tasks.SetFacesToCullTask;
 
 /**
  * TODO: Add javadocs
  */
 public final class SetFacesToCull implements StateChange {
-
     private static SetFacesToCull defaultInstance = new SetFacesToCull(GL_BACK); // also specified in OpenGL documentation
-    private SetWhatFacesToCullTask task;
+    private static Map<Integer, String> modeNameMap = ImmutableMap.of(GL11.GL_BACK, "GL_BACK",
+            GL11.GL_FRONT, "GL_FRONT",
+            GL11.GL_FRONT_AND_BACK, "GL_FRONT_AND_BACK");
```

Comment:
In the constructor you use GL_BACK and the other constants without the GL11 qualifier. I recommend to remove it from here too, if possible. Or, if you prefer, to use GL11 also below.